src/main/java/uk/co/ramp/covid/simulation/covid/Covid.java

```diff
@@ -98,17 +106,18 @@ public boolean isDead() {
     
     private void setSymptomatic() {
         symptomaticCase = pSymptoms.sample();
+        if(ccase.getAge() >= 60) symptomaticCase = pSymptomsPensioner.sample(); // This is set to 60 because the probability is from the Diamond Princess where people were aged > 60 
```

Comment:
Do we want a parameter for the pensioner age?

Again, possibly something we want to add in future if it seem the Diamond Princess is the best data we currently have.

Reply:
Well if we did then pensioners hsould be 65, because in this model it mostly refers to working age and working age in Scotland goes to 65, just happens that these data concern a cut-off of 60